### DISCUSSION TITLE
Getting 'target' is a directory errors

### DISCUSSION THREAD

**Discussion by cptaps** [wildfly/wildfly-maven-plugin#580] [General]
When the wildfly-maven-plugin is configured with both a feature-pack element and the discover-provisioning-info element we get an error about a 'target' being a directory.

This appears to come from [plugin/src/main/java/org/wildfly/plugin/common/Utils.java](https://github.com/wildfly/wildfly-maven-plugin/blob/b9441715382be52e3fb1e0c0a4d0a8311fe9b143/plugin/src/main/java/org/wildfly/plugin/common/Utils.java#L136) where a directory is being passed into an API that requires a File.

**Comment by jfdenise** [comment]
Hi, thank-you for reporting this problem, I have logged: https://issues.redhat.com/browse/WFMP-277. The fix PR: https://github.com/wildfly/wildfly-maven-plugin/pull/581
Closing the discussion.
